en/index.json

```diff
@@ -326,9 +326,9 @@
       "label": "AMD"
     }
   },
-  "leaderboard" : {
-    "global": "Global",
-    "my_supporters": "My Supporters",
-    "go_to_leaderboard": "Go to Leaderboard"
+  "leaderboards" : {
```

Comment:
I think this should be added to the crypto repo then pull the build langs in crypto miner? https://github.com/anyTV/freedom-crypto-miner-i18n

Do git submodule init and update after that.